_i18n/en.yml

```diff
@@ -501,6 +501,11 @@ faq:
   avuln: >
     The Monero community has created a series of videos called "Breaking Monero", where potential Monero vulnerabilities are explored and discussed. There are 14 videos, with each exploring a different subject. Check out <a href="https://www.monerooutreach.org/breaking-monero/">the videos on monerooutreach.org</a>.
   vulnspotify: Available on Spotify as podcast
+  qasicresistance: "What is ASIC resistance? Why is it important?"
+  aasicresistance: ASICs are basically special computers created to do only one job, contrary to normal computers, which are made for multiple purposes. This characteristic makes ASICs very efficient for @mining.
+  aasicresistance1: The problem is that these devices are very expensive, consume a lot of electricity and can be afforded by few, wealthy people. This leads to few entities owning a big amount of the hashrate of the network, which is a serious threat to the security of the network itself. For example, if big ASIC operators collude and manage to gain the majority of the hashrate of the network, they could arbitrarily reject transactions.
+  aasicresistance2: >
+    Monero fixes this problem by being ASIC-resistant: it uses an algorithm (@randomx) that makes impossible for ASICs to mine on the network. Miners must use common consumer hardware, which forces them to compete fairly. The Monero network is currently protected by thousands of computers. This results in a network much harder to attack, no miner having significant advantage over other miners (they all use more or less the same hardware) and less energy consumption.
```

Comment:
```suggestion
    Monero fixes this problem by implementing an ASIC-resistant proof-of-work algorithm: it uses @randomx, an algorithm that strongly reduces the efficiency of ASICs mining, disincentivizing the development of this kind of hardware. Miners are economically incentivized to use common consumer hardware. As a result, the Monero network is currently protected by thousands of computers mining its algorithm using “regular” computers equipped with CPUs. This results in a network much harder to attack, since no miner has significant advantage over other miners (they all use hardware with similiar efficiency).
```

Reply:
Thank you for your reviews. My issue is that you tend to suggest complete rewords more than fixes or suggestions in your reviews. I'm not a big fan of this type of review, because if something is wrong i'm happy to correct it and if something is not explained well i'm happy to add suggested clarifications. But i don't agree with rewording an entire paragraph according to the style or wording preference of another contributor, if it's a one time thing it's fine, but it seems to be the norm in your reviews.

So, i will add some clarification where needed but i won't change it completely.

Reply:
I added part of your suggestions.

Reply:
You don't need to feel any pressure to accept or reject my suggestions. I'm not trying to impose my ideas or attack your contribution in any form, I'm just writing some additional information that you may find useful. Feel free to pick what you find interesting and discard what you think is unimportant.